index.html

```diff
@@ -238,7 +221,7 @@ <h2 class="contact-us">contact us</h2>
 			<textarea name="message" class="message" maxlength="500" placeholder="Enter text here." required></textarea>
 
 			<button type="submit" class="see-project">
-				see project
+				Send
 			</button>
```

Comment:
- [ ] Buttons are multi-state elements and it is very important to provide some visual feedback to users when they interact with buttons on your page. Currently, the `.see-project` button is missing two things:
- Template style for the `pressed` state
- Template style (*correct box-shadow with no border property*) for the `hover` state

Although you added some rules for the hover state, it would be better to implement the style exactly as it is from the template. For developers, being able to correctly *parse* and *replicate* templates from designers is a valuable skill.  Kindly implement these so that you can have the required visual interaction with the buttons 👍🏽

Reply:
done, thanks!